### PR TITLE
New runcard for a subset of spinq10q qubits (q6 to q10)

### PR DESCRIPTION
**Pull request by aorgazf**
Hi All,
This PR adds a new platform to be able to use a subset of spinq10q qubits. The qubits available are qubits 6 to 10.
Please note that the fluxlines of qubits 7 & 8 are not working well and require more power to bias the qubits; and the fluxlines of qubits 9 and 10 do not work at all.
Single qubit gates are not fully calibrated. I will continue to work on them.

### CODE REVIEW

File: spinq10q_610_qblox.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    instruments = {
+        controller.name: controller,
+        twpa_pump1.name: twpa_pump1,
+    }
+    instruments.update(modules)
+    instruments = load_instrument_settings(runcard, instruments)
+
+    # DEBUG: debug folder = report folder
+    import os
+    from datetime import datetime
```
Comment (andrea-pasquale) [inline spinq10q_610_qblox.py:67]:
```suggestion
```

File: spinq10q_610_qblox.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        qubits[q].drive = channels[f"L6-{q}"]
+        qubits[q].flux = channels[f"L6-{38+q}"]
+        channels[f"L6-{38+q}"].qubit = qubits[q]
+        # set maximum allowed bias
+        qubits[q].flux.max_bias = 2.5
+
+    settings = load_settings(runcard)
+
+    return Platform(
+        "spinq10q_610_qblox", qubits, pairs, instruments, settings, resonator_type="2D"
```
Comment (andrea-pasquale) [inline spinq10q_610_qblox.py:116]:
```suggestion
        "spinq10q_610", qubits, pairs, instruments, settings, resonator_type="2D"
```
Generally I would avoid putting the electronics in the name of the platform or the file. If there is a change in control electronics we should remove/add new runcard/platform.py file everytime.

**Comment by andrea-pasquale** [comment]
@PiergiorgioButtarini I tried to run the bot but the following error is raised
https://github.com/qiboteam/qibolab_platforms_qrc/actions/runs/7297606755/job/19887141149?pr=100#step:8:25
Whenever you have time could you please help @aorgazf to update this code to the latest version of the driver?

**Comment by PiergiorgioButtarini** [comment]
> @PiergiorgioButtarini I tried to run the bot but the following error is raised https://github.com/qiboteam/qibolab_platforms_qrc/actions/runs/7297606755/job/19887141149?pr=100#step:8:25 Whenever you have time could you please help @aorgazf to update this code to the latest version of the driver?

Now is solved!

**Comment by andrea-pasquale** [comment]
> > @PiergiorgioButtarini I tried to run the bot but the following error is raised https://github.com/qiboteam/qibolab_platforms_qrc/actions/runs/7297606755/job/19887141149?pr=100#step:8:25 Whenever you have time could you please help @aorgazf to update this code to the latest version of the driver?
> 
> Now is solved!

Thank you!

**Pull request closed without merging**